README.md

```diff
@@ -146,6 +146,10 @@ For a debug build:
 Test
 ----
 
+If a ruby was found during configuration, execute the following command before running tests:
```

Comment:
We could, but iirc the test target is like the install target, a bit special and difficult to make it depend on things.